src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -111,13 +111,23 @@ export class NetworkStorage {
     const patterns: Protocol.Fetch.RequestPattern[] = [];
 
     for (const value of this.#interceptMap.values()) {
-      for (const urlPatternSpec of value.urlPatterns) {
-        const urlPattern: string =
-          NetworkStorage.cdpFromSpecUrlPattern(urlPatternSpec);
-        for (const phase of value.phases) {
+      for (const phase of value.phases) {
```

Comment:
Why do we introduce this new loop?

Reply:
We did not we switched the order of the loops as `Phase` is always at least one and `UrlPatterns` may be empty.